library/Businessprocess/Modification/NodeAction.php

```diff
@@ -144,7 +144,7 @@ public function serialize()
      */
     public static function unSerialize($string)
     {
-        $object = json_decode($string, JSON_FORCE_OBJECT);
+        $object = json_decode($string, true);
```

Comment:
should be false, I guess?

Reply:
JSON_FORCE_OBJECT = `16`, in bool `true`